compute/admin_guide/install/install-gke.adoc

```diff
@@ -44,7 +45,7 @@ $ kubectl create clusterrolebinding your-dev-user-cluster-admin-binding \
     --user=your-dev-user@your-org.iam.gserviceaccount.com
 ----
 
-. With the *Kubernetes Engine Developer* service account, continue with the standard Kubernetes install procedure for Prisma Cloud Console and Defenders, <<_install_console,starting here>>.
+. With the *Kubernetes Engine Developer* service account, continue with the standard Kubernetes install procedure for the xref:install_kubernetes.adoc#install_console[Prisma Cloud Console] and xref:./install_defender/install_cluster_container_defender.adoc#deamonset-gke[Defenders],
```

Comment:
@rcaballeromx I'm not sure this is the right place to add this instructions since the next steps instract the customers to deploy the defender using "Go to Manage > Defenders > Manage > DaemonSets." and not be creating the `daemonset.yaml` file. 
The reason that we need the `daemonset.yaml` file is because the customer needs to edit the file according to the GKE instructions for the multi-arch cluster.
The only place I could find those instructions is here [Deploying Defender DaemonSets](https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/install/install_kubernetes) but it also includes just the twistcli installation and not the UI (I couldn't find the UI installations).

My comment about specifying that it's also required when using just ARM nodes is relevant for this place as well

Reply:
@MorShabiPAN We do have [UI instructions for GKE](https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/install/install_defender/install_cluster_container_defender).
The instructions don't mention the `deamonset.yaml` file because customers didn't need to edit it before.
We have several options to let customers know that they need to edit it now for multi-architecture and ARM, but we still need to keep these cross-references.
For now, I have added a note on those instructions letting customers know they need to edit the file. PTAL

Reply:
I've added the note as we discussed but that needed to be done in the included fragment file.